__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const { topicData, userData, articleData, commentData } = require('../db/data/test-data/');
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe('Test /api/topics endpoint', () => {
+    test('Test connection with /api/topics, should receive 200 as the status code to show connection', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)                  
+    })
```

Comment:
If you change your test descriptions to show what kind of request you are doing for example - GET api/topics instead of test then when you have lots of tests it will be easier to find which ones are failing 👀